### PR TITLE
(Fixes #3719) Make installing kn into a re-usable snippet 

### LINKED ISSUE #3719: Create re-usable snippet for installing `kn`
We want to poke at out our emerging re-usable content strategy by getting this low-hanging fruit delivered

### PR DESCRIPTION
**Pull request by omerbensaadon**

### CODE REVIEW

File: docs/snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Snippets
```
Comment (julz) [inline docs/snippets/README.md:1]: might be worth putting the `snippets` dir in `base_paths` so the path doesn't need to be specified (see https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#specifying-snippet-locations). Also if the snippets dir lives under docs/ we should probably add it to [exclude](https://github.com/knative/docs/blob/mkdocs/mkdocs.yml#L276) so it doesn't get built.

File: docs/snippets/install-kn.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        - <a href="https://gcr.io/knative-releases/knative.dev/client/cmd/kn" target="_blank">Latest release</a>
+
+        You can run `kn` from a container image. For example:
+
+        ```
+        docker run --rm -v "$HOME/.kube/config:/root/.kube/config" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
+        ```
+
+        !!! note
```
Comment (abrennan89) [inline docs/snippets/install-kn.md:49]: One super minor nit but I'd take this out of the note and put it before the numbered steps in this bit, just cos it's important and also a note inside a tab like this looks kinda odd. Your call though.

**Review by abrennan89 — COMMENTED** [review]
One minor suggestion but this is 🔥 overall, thanks @omerbensaadon !

File: docs/snippets/install-kn.md (hunk 2 above)
Comment (abrennan89) [reply docs/snippets/install-kn.md:49]: This isn't a blocker so I'll let @julz lgtm cos it looks like he had suggestions also

File: docs/snippets/README.md (hunk 1 above)
Comment (omerbensaadon) [reply docs/snippets/README.md:1]: I will do this now thank you!!

**Comment by omerbensaadon** [comment]
found the problem, testing on GH pages and pushing fix

**Comment by omerbensaadon** [comment]
ok this works!

**Review by julz — APPROVED** [review]
/lgtm

**Pull request merged**